### PR TITLE
Fix toolbar

### PR DESCRIPTION
**Pull request by osmovia**

### CODE REVIEW

File: app/src/main/java/com/example/learingrealmandretrofit/api/ApiInterface.kt [hunk 1]
```diff
@@ -16,76 +17,102 @@ interface ApiInterface {
...
 
     @POST("sessions")
     fun signIn(
-        @Body params: SignInRequest):  Call<AuthenticationResponse>
+        @Body params: SignInRequest
+    ):  Call<AuthenticationResponse>
 
     @DELETE("sessions/{token}")
     fun signOut(
         @Header(tokenKey) tokenSession: String,
```
Comment (skliarov) [inline app/src/main/java/com/example/learingrealmandretrofit/api/ApiInterface.kt:30]: You need to refactor this header, it should be inserted automatically

File: app/src/main/java/com/example/learingrealmandretrofit/objects/CardDeckParameters.kt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package com.example.learingrealmandretrofit.objects
+
+data class CardDeckParameters(
+    val id: Int = 0,
+    val cardId: Int,
```
Comment (skliarov) [inline app/src/main/java/com/example/learingrealmandretrofit/objects/CardDeckParameters.kt:5]: Why `id` has default value, but `cardId` and `deckId` do not?

File: app/src/main/java/com/example/learingrealmandretrofit/fragment/TabsFragment.kt [hunk 3]
```diff
@@ -22,12 +26,21 @@ class TabsFragment : Fragment(R.layout.fragment_tabs) {
...
+
         NavigationUI.setupWithNavController(binding.bottomNavigationView, navController)
         val appBarConfiguration = AppBarConfiguration(
             setOf(R.id.deckFragment, R.id.cardFragment, R.id.settingFragment)
         )
-        setupActionBarWithNavController(
-            (requireActivity() as MainActivity), navController, appBarConfiguration
-        )
+
+//        setupActionBarWithNavController(requireActivity() as MainActivity)
```
Comment (skliarov) [inline app/src/main/java/com/example/learingrealmandretrofit/fragment/TabsFragment.kt]: Commented out code

File: app/src/main/java/com/example/learingrealmandretrofit/deck/viewmodel/DeckViewModel.kt [hunk 4]
```diff
@@ -57,38 +56,25 @@ class DeckViewModel(private val token: String) : ViewModel() {
...
+                val persistedDeck = realmTransaction.where(Deck::class.java).equalTo("id", deck.id).findFirst()
+
+                if (persistedDeck == null) {
+                    val newDeck = Deck(id = deck.id, title = deck.title)
+                    realmTransaction.insert(newDeck)
+                }
+            }
+        }, {
+            realm.close()
+//            BaseApi.retrofit.getCardDecks(...)
```
Comment (skliarov) [inline app/src/main/java/com/example/learingrealmandretrofit/deck/viewmodel/DeckViewModel.kt:73]: Commented out code

File: app/src/main/java/com/example/learingrealmandretrofit/card/viewmodel/AddCardToDeckViewModel.kt [hunk 5]
```diff
@@ -51,46 +50,78 @@ class AddCardToDeckViewModel(private val token: String, private val cardId: Int)
...
+                }
             }
 
             override fun onFailure(call: Call<CardDeckResponse?>, t: Throwable) {
-                TODO("Not yet implemented")
+                _showSpinner.value = false
+                _showToast.value = R.string.connection_issues
             }
         })
 //        _showSpinner.value = true
```
Comment (skliarov) [inline app/src/main/java/com/example/learingrealmandretrofit/card/viewmodel/AddCardToDeckViewModel.kt:127]: Commented out code

**Comment by osmovia** [comment]
Thanks for being very quick feedback.

**Pull request merged**